tests/numerics/ad/test_secondary_operator.py

```diff
@@ -0,0 +1,300 @@
+"""Test module to test the functionality of the secondary operator class, with
+external evaluation, storage and access of values and derivative values."""
+
+from __future__ import annotations
+
+from typing import Callable
+
+import numpy as np
+import pytest
+
+import porepy as pp
+
+VAR1_NAME = "var1"
+VAR2_NAME = "var2"
+INTFVAR_NAME = "intfvar"
+
+
+@pytest.fixture
+def eqsys() -> pp.ad.EquationSystem:
+    """Fixture containing the equation system, variables and MDG for testing the
+    secondary operator.
+
+    Unit square with 2 line fractures forming a cross in the middle.
+
+    2 Variables on subdomains, 1 variable on interfaces.
+
+    """
+
+    bounding_box = {"xmin": 0, "xmax": 1, "ymin": 0, "ymax": 1}
+    domain = pp.Domain(bounding_box=bounding_box)
+
+    frac1 = pp.LineFracture(np.array([[0.5, 0.5], [0.2, 0.8]]))
+    frac2 = pp.LineFracture(np.array([[0.2, 0.8], [0.5, 0.5]]))
+
+    network_2d = pp.create_fracture_network([frac1, frac2], domain)
+
+    mesh_args: dict[str, float] = {"cell_size": 0.1, "cell_size_fracture": 0.1}
```

Comment:
```suggestion
    mesh_args: dict[str, float] = {"cell_size": 0.5, "cell_size_fracture": 0.5}
```
I think this should work, and the test will run faster.